### ISSUE TITLE
3.6.10 breaks backwards compatibility

### ISSUE THREAD

**Issue by lukasNo1** [putyourlightson/craft-blitz#261]
[3.6.10](https://github.com/putyourlightson/craft-blitz/commit/4d680b0f2b263d922510cd2a2d37ecbdcd7eb8b0#diff-7c22079a785867f2b5a0b9a453c977b0R318)
(/src/services/CacheRequestService.php:318)

This commit broke the backwards compatibility and took my website down.

It introduced a change where forward slashes now get escaped. I already escape these slashes in my pattern though.

Which lead to a `preg_match(): Unknown modifier ')'` error

```
 'excludedUriPatterns'     => [
            [
                'siteId'     => '',
                'uriPattern' => '(.*\/)?dcn(\/.*)?',
            ],
        ],
```

I have now removed the backslashes in the pattern for it to work again. For backwards compatibility you should check if there are backslashes present in the pattern. I think more people can run into this error.

**Comment by bencroker** [comment]
Hmm sorry about that. The backslashes were not required in previous versions either though.

**Comment by lukasNo1** [comment]
Hmm yes thats true. Valid PCRE patterns shouldn't throw errors though.
Maybe mention it in the release notes or only replace unescaped forward slashes with preg_replace for example. (Look at performance impact for this one though)

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-blitz/commit/2f467a3c46ac5dda9c6cf56de350d1d172614d2c and released in 3.6.11.
